ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -329,6 +331,15 @@ public class ClusterMapConfig {
   @Default("false")
   public final boolean clustermapEnableDeleteInvalidDataInMysqlAggregationTask;
 
+  /**
+   * The possibility to delete invalid hosts in mysql aggregation tasks. It should be a value between 0 and 1.
+   * When it's 0, it means we are never going to delete invalid hosts in the aggregation task. When it's 1, it
+   * means every time we run an aggregation task, we will try to delete the invalid hosts.
+   */
+  @Config(DELETE_INVALID_HOSTS_IN_MYSQL_AGGREGATION_TASK_POSSIBILITY)
+  @Default("0.01")
+  public final float clustermapDeleteInvalidHostsInMysqlAggregationTaskPossibility;
```

Comment:
I was just thinking if we could also remove invalid hosts by listening to changes in cluster map. For example, a) add a method `onHostRemoved(String hostname)` in `ClusterMapChangeListener` and b) StatsManager could register with the listener with clusterMap. Would it work?